src/index.tsx

```diff
@@ -40,6 +43,16 @@ window.addEventListener('error', error => {
   })
 })
 
+const SENTRY_DSN: string | undefined = process.env.REACT_APP_SENTRY_DSN
```

Comment:
Could you put the needed values for Sentry to env.production?

Reply:
they are not secret so I think it's the easiest way to go

Reply:
Added the value